tdrs-backend/tdpservice/reports/views.py

```diff
@@ -21,11 +21,35 @@ class ReportFileViewSet(ModelViewSet):
     """Report file views."""
 
     http_method_names = ['get', 'post', 'head']
+    filterset_fields = ['year', 'quarter']
```

Comment:
This addition allows the UI to apply query parameters to the list endpoint to show only reports for the selected year and quarter.